SPECS/libseccomp/libseccomp.spec

```diff
@@ -62,6 +62,9 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %{_mandir}/man3/*
 
 %changelog
+* Thu Dec 05 2024 Nan Liu <liunan@microsoft.com> - 2.5.5-1
```

Comment:
Wondering if we should target `3.0-dev` instead ?

Reply:
Thanks Falak! I created a new one targeting 3.0-dev: https://github.com/microsoft/azurelinux/pull/11383. Closing out this PR.